src/tests/unit/test_tree_node_helpers.py

```diff
@@ -1,1087 +0,0 @@
-import unittest
```

Comment:
stuff in this file has been moved to the folder "test_tree_functionality and splitted up into several files